cmd/libs/go2idl/go-to-protobuf/protobuf/generator.go

```diff
@@ -258,6 +258,7 @@ func (b bodyGen) doStruct(sw *generator.SnippetWriter) error {
 					options = append(options, fmt.Sprintf("%s = %s", key, v))
 				}
 			}
+		// TODO: What's that? What's the purpose of it?
```

Comment:
ohh - actually it is being used in the current code:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/resource/quantity.go#L88

So leaving this here.